Makefile.builder

```diff
@@ -10,6 +10,10 @@ ifeq ($(PACKAGE_SET),vm)
   ARCH_BUILD_DIRS := archlinux
 endif
 
+ifeq ($(DIST),trusty)
+  $(shell sed -i /locales-all,/d  $(CHROOT_DIR)/$(DIST_SRC)/debian/control)
+endif
+
```

Comment:
This is wrong place - `$(shell ...)` will be executed every time `Makefile.builder` gets parsed. You should place it in `source-debian-quilt-copy-in` target, and without `$(shell ...)`.